core/src/main/java/org/infinispan/topology/CacheTopologyControlCommand.java

```diff
@@ -36,8 +36,8 @@
       JOIN,
       // A member is signaling that it wants to leave the cluster.
       LEAVE,
-      // A member is confirming that it finished the rebalance operation.
-      REBALANCE_CONFIRM,
+      // A member is confirming that it has finished the topology change during rebalance.
+      REBALANCE_PHASE_CONFIRM,
```

Comment:
1. To me, a topology change is just installing a new topology, it doesn't include requesting or receiving new state.
2. Saying "*the* topology change during rebalance" implies that only one topology change is being confirmed per rebalance.